lib/paper_trail/config.rb

```diff
@@ -31,6 +32,7 @@ def initialize
 
       # Variables which affect all threads, whose access is *not* synchronized.
       @serializer = PaperTrail::Serializers::YAML
+      @has_paper_trail_defaults = {}
```

Comment:
Should we use `HashWithIndifferentAccess`?

Reply:
Sure - makes sense to me.